### DISCUSSION TITLE
Blank page when launching a project

### DISCUSSION THREAD

**Discussion by anti-social** [DioxusLabs/dioxus#1141] [Q&A] [answered]
OS: `NixOS 23.05`
Dioxus version: `0.3.2`

Cannot understand why I got blank page whatever I do. A strange think I'm watching is a version when I launch `dioxus serve` command:

```
Dioxus @ v0.1.5 [15:07:25] 
```

But when I launch `dioxus --version`it prints:

```
dioxus 0.3.2
```

Another strange thing is when I build a project, produced `dist/index.html` does not contain any scripts and links.

I also tried to launch a `todomvc` example but got the same result.

**Comment by ealmloff** [comment, marked as answer]
In the published version of the Dioxus CLI, you need to include some code to start the wasm from javascript in your index.html: https://github.com/DioxusLabs/dioxus/blob/4c37d5c2100d012b954603b9d916a1482c3773b7/packages/cli/src/assets/index.html#L12

**Reply by anti-social** [reply]
It works, thanks. I'll try to update examples
